src/context.js

```diff
@@ -0,0 +1,25 @@
+import ChannelPool from './channel-pool.js'
```

Comment:
我看到你大部分的 code 都有加 `;` 那麼這裡也加上吧。
```suggestion
import ChannelPool from './channel-pool.js';
```